third_party/move/move-compiler-v2/tests/ability-check/v1-typing/instantiate_signatures.exp

```diff
@@ -129,3 +129,12 @@ error: type `R` is missing required ability `drop`
    │
    = required by instantiating type parameter `T:drop` of struct `S`
    = required by instantiating type parameter `T:drop` of struct `S`
+
+warning: Unnecessary trailing semicolon
+   ┌─ tests/ability-check/v1-typing/instantiate_signatures.move:45:40
+   │
+45 │         S<S<(u64, u64)>> { f: abort 0 };
```

Comment:
V1 does not generate this error: https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-compiler/tests/move_check/typing/instantiate_signatures.exp